### PR TITLE
Provide a programmatic / API driven way to create PlanetScale DB and connection string

### PR DESCRIPTION
**Pull request by jonico**
So far, beam's README describes the steps to manually [create a PlanetScale database](https://docs.planetscale.com/tutorials/planetscale-quick-start-guide#create-a-database) and [connection string](https://docs.planetscale.com/concepts/connection-strings#creating-a-password).

This PR contributes
*  scripts to create a database and connection string using the [pscale CLI](https://github.com/planetscale/cli) including instructions on how to set the Prisma connection string properly
*  a GitHub Action workflow that can be run from GitHub to create the database and connection string using Web based authentication
image

* a description in the README file how to use the scripts or the Action workflow

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,17 @@ npm install
 
 - [Create a PlanetScale database](https://docs.planetscale.com/tutorials/planetscale-quick-start-guide#create-a-database)
 - Create a [connection string](https://docs.planetscale.com/concepts/connection-strings#creating-a-password) to connect to your database. Choose **Prisma** for the format
+- Alternatively, create your PlanetScale database and connection string programmatically, either with a script using [pscale CLI](https://github.com/planetscale/cli) or by running a GitHub Action wokflow:
```
Comment (jasonlong) [inline README.md:19]:
```suggestion
- Alternatively, create your PlanetScale database and connection string programmatically, either with a script using [pscale CLI](https://github.com/planetscale/cli) or by running a GitHub Action workflow:
```

**Comment by jasonlong** [comment]
Thanks @jonico. This is nit-picky, but would you be able to take a screenshot from a narrower browser window? It'll be easier to see what's going on not being so wide.

**Comment by jonico** [comment]
> It'll be easier to see what's going on not being so wide.

Absolutely, will do

**Comment by jonico** [comment]
@jasonlong: Updated the screenshot in the README and in the PR description, let me know what you think

**Comment by jasonlong** [comment]
Thanks @jonico, that fits in much better. ✨ 

I think I'd like to move this new README section to its own page as we do for some of the other instructions just to keep the main instructions streamlined. Let's go ahead and merge this and I'll do that in a follow-up PR.

Thanks again!

**Pull request merged**